packages/aws-cdk-lib/cx-api/lib/features.ts

```diff
@@ -1157,6 +1158,21 @@ export const FLAGS: Record<string, FlagInfo> = {
     recommendedValue: true,
     introducedIn: { v2: '2.160.0' },
   },
+
+  //////////////////////////////////////////////////////////////////////
+  [USE_CORRECT_VALUE_FOR_INSTANCE_RESOURCE_ID_PROPERTY]: {
+    type: FlagType.BugFix,
+    summary: 'When enabled, the value of property `instanceResourceId` in construct `DatabaseInstanceReadReplica` will be set to the correct value which is `DbiResourceId` instead of currently `DbInstanceArn`',
+    detailsMd: `
+      Currently, the value of the property 'instanceResourceId' in construct 'DatabaseInstanceReadReplica' is not correct, and set to 'DbInstanceArn' which is not correct when it is used to create the IAM Policy in the grantConnect method.
+      
+      When this feature flag is enabled, the value of that property will be as expected set to 'DbiResourceId' attribute, and that will fix the grantConnect method.
+    `,
+    introducedIn: { v2: 'V2NEXT' },
+    defaults: { v2: false },
```

Comment:
I don't think this line is needed. It's only required if you need to set the default to true, otherwise it should be false automatically.